tests/setup.js

```
@@ -0,0 +1,99 @@
+		window?.localStorage?.clear();
+	} catch (_e) { /* ignore */ }
+	try {
+		global?.localStorage?.clear();
+	} catch (_e) { /* ignore */ }
+	vi.clearAllMocks();
+	// default umami stub
+	if (typeof window !== 'undefined') {
+		window.umami = window.umami || { track: vi.fn() };
+		window.umami.track = vi.fn();
+	}
+	globalThis.window = globalThis.window || window || {};
```

ESLint

[error] 'globalThis' is not defined.

(no-undef)